### PR TITLE
Doc cleanup [20190718]

### PR DESCRIPTION
**Pull request by levitte**
## util/find-doc-nits: fixups

- Treat .pod.in files as well, and parse out the base name for those
  too.
- Correct the detection of the description part in the NAME section
  (the separating dash MUST be preceeded with a space)
- Allow slahes in names of the NAME section (convert them to dashes
  for file name comparison).  This allows manual pages for some of our
  header files, such as openssl/core.h.
- Properly detect repeated names in the NAME section.

## doc: fix some links

Some links are aged and need an adjustment.

### CODE REVIEW

File: util/find-doc-nits [hunk 1]
```diff
@@ -577,7 +582,7 @@ if ( $opt_c ) {
     # See if each has a manpage.
     foreach ( @commands ) {
         next if $_ eq 'help' || $_ eq 'exit';
-        if ( ! -f "doc/man1/$_.pod" ) {
+        if ( ! -f "doc/man1/$_.pod"  ) {
```
Comment (mattcaswell) [inline util/find-doc-nits]: Why this?

**Review by mattcaswell — APPROVED** [review]
LGTM. One minor nit.

File: util/find-doc-nits (hunk 1 above)
Comment (levitte) [reply util/find-doc-nits]: No reason. Oops. Or to see if I have your attention. Your pick :wink:

**Comment by levitte** [comment]
Merged.

3cb45a55853db05d5af1b564a55491a22f592305 doc: fix some links
f6800e37b762563e79115ebdb233c0c07afc23e5 util/find-doc-nits: fixups

**Pull request closed without merging**